Tests/UpstreamIntegrationTests/UpstreamIntegrationTests.swift

```diff
@@ -98,7 +95,7 @@ class UpstreamIntegrationTests: TestBase {
             {
               "ttlSeconds" : 1800,
               "scope" : "EdgeNetwork",
-              "hint" : "or2"
+              "hint" : "stringType"
```

Comment:
Add a comment to explain our expectation and weak match concept.

Reply:
The same applies for ttlSeconds as we only do exactMatchPaths on scope. Should we update the note to check exactMatchPaths for the keys that will do exact matching vs type matching?